docs/howto/start-agent-remotely.md

```diff
@@ -0,0 +1,73 @@
+# How to start the agent remotely
+
+Ubuntu Pro for WSL, being a Microsoft Store application, cannot ship user services as of the time of this writing (late
+2024), but can deploy startup tasks instead, programs that run with user permissions when the said user logs on the
+Windows device. The UP4W background agent runs as a startup task. By design a startup task is only enabled by the
+operating system when the user interacts with the application for the first time. While this behaviour is a feature for
+end users it presents a source of friction for deployments at scale, when system administrators expect zero-touch
+deployment of UP4W to just work.
```

Comment:
This material seems broadly duplicated from the other guide.
While I understand it is relevant in both cases, I think it would be preferable if this more explanatory material, which provides valuable context for the guide(s), instead be captured in a dedicated Explanation article.

That explanation article could then be referenced in these guides, and any future guides pertaining to this topic, to provide context. As it stands, I think this explanatory material upsets the flow of the guides.

Consider truncating this if possible and we can try to encapsulate this in an explanation section next cycle.